apps/antalmanac/src/components/Map/Map.tsx

```diff
@@ -249,9 +251,14 @@ export default function CourseMap() {
                 {/* Draw a marker for each class that occurs today. */}
                 {markersToDisplay.map((marker, index) => {
                     // Find all courses that occur in the same building prior to this one to stack them properly.
+                    // TODO Handle multiple buildings between class comparisons on markers.
```

Comment:
Is this referring to the fact that we're only using `marker.bldg[0]` here and ignoring all others if they don't share the same `building` attribute?

Reply:
I don't remember making this comment, it might've been Alan. But yeah, we can only take one building because that's how the routing will calculate the route. When we show the routes, we show how to get from class 1 to class 2, class 2 to class 3, etc. and we don't have a way to represent class 1 to multiple locations for class 2.